test/attr.test.js

```diff
@@ -11,6 +11,16 @@ class Fixture extends React.Component {
 const it = createTest(<Fixture />)
 
 describe('#attr', () => {
+  it('fails when actual is not a enzymeWrapper', () => {
+    expect(() => {
+      expect(undefined).to.have.attr('key', 'somekey')
+      expect({ 'foo': 'bar' }).to.have.attr('key', 'somekey')
```

Comment:
To avoid repeating yourself and fixed the note above you can do:

```js
it('fails when the actual is not an enzyme wrapper'), () => {
  [undefined, { foo: 'bar' }, [], 'test', 12345].forEach((actual) => {
    expect(() => {
      expect(actual).to.have.attr('key', 'somekey')
    }).to.throw()
  })
})
```